### PR TITLE
Breed/update readme

### PR DESCRIPTION
**Pull request by BRReed**
README update for first iteration of expressjs intro project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # hello-relativepath
-introduction to express.js
+
+
+## Introduction
+
+The purpose of the first iteration of this project is to run a basic express.js API. 
+
+In order to run this first iteration you must have nodejs and express installed.
```
Comment (stephenlauck) [inline README.md]: A script should do this.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+## Introduction
+
+The purpose of the first iteration of this project is to run a basic express.js API. 
+
+In order to run this first iteration you must have nodejs and express installed. 
+
+## Install Dependencies
+
+1. npm `pamac install npm`
```
Comment (stephenlauck) [inline README.md]: A script should do this. In the README should be the command to run the script.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+
+
+## Deploy
+
+1. Download `hello-world.js` file.
```
Comment (stephenlauck) [inline README.md]: This is what the script should do and be replaced with how to run the script.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+2. Using a terminal navigate to the directory `hello-world.js` is in.
+3. Enter `npm install express`. You can append `--no-save` flag to install express temporarily.
+4. Enter `node hello-world.js`
+
+## Confirm
+
+1. Open your favorite web browser
+2. Enter `localhost:3000` into the address bar.
```
Comment (stephenlauck) [inline README.md]: This should be done via a script.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,31 @@
...
-introduction to express.js
+
+
+## Introduction
+
+The purpose of the first iteration of this project is to run a basic express.js API. 
+
+In order to run this first iteration you must have nodejs and express installed. 
+
+## Install Dependencies
```
Comment (stephenlauck) [inline README.md]: :+1:

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (BRReed) [reply README.md]: README shrunk down, now Deploy is copy a command and paste it into your shell.

Once I upload the hello-world-deploy.sh (after we get through the README) I can point that command to an actual path.

side note: I wrote and tested all of the shell script in a docker container already (and uploaded it to my server so I could confirm the readme command works).

File: README.md [hunk 6]
```diff
@@ -3,26 +3,16 @@
...
-
-## Install Dependencies
-
-1. npm `pamac install npm`
+## Deploy
 
+Copy and paste the following command into your shell:
 
-## Deploy
+`sh <(curl -s http://<path to hello-world-deploy.sh)`
```
Comment (stephenlauck) [inline README.md]: Might be better to just have a scripts/ directory in the repo and just call them from there. For example

`bash scripts/app_deploy.sh`

File: README.md [hunk 7]
```diff
@@ -7,9 +7,12 @@ The purpose of the first iteration of this project is to run a basic express.js
 
 ## Deploy
 
-Copy and paste the following command into your shell:
+1. Download the repository to a directory of your choosing.
+
+2. Navigate to the repo directory in your shell.
+
+3. Enter the command `sh hello-world-deploy.sh` into your shell.
```
Comment (stephenlauck) [inline README.md]: :+1: 

Also like that you're keeping everyting POSIX compatible and using just sh.

**Review by stephenlauck — APPROVED** [review]

**Review by stephenlauck — APPROVED** [review]

**Pull request merged**